src/SSW.SophieBot/components/SSWSophieBot.HttpClientComponents.PersonQuery/EmployeesHelper.cs

```diff
@@ -159,5 +160,26 @@ private static bool IsProjectNameEqual(string sourceProjectName, string original
             var prjName = _rgx.Replace(sourceProjectName, string.Empty);
             return _rgx.Replace(originalProjectName, string.Empty).IndexOf(prjName, StringComparison.OrdinalIgnoreCase) >= 0;
         }
+
+        public static List<GetEmployeeModel> filterEmployees(List<GetEmployeeModel> employees)
```

Comment:
```suggestion
        public static List<GetEmployeeModel> FilterEmployees(List<GetEmployeeModel> employees)
```
Method names in C# need to be Camel cases